.github/workflows/test.yaml

```diff
@@ -170,6 +175,57 @@ jobs:
       - name: Report
         id: report
         run: echo "outcome=$([ ${{ steps.metadataYaml.outputs.exists }} = 'true' ] && echo success || echo skip)" >> $GITHUB_OUTPUT
+  tox-lint:
+    name: Lint tox.ini
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: ${{ inputs.working-directory }}
+    outputs:
+      outcome: ${{ steps.report.outputs.outcome }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run lint
+        run: |
+          # Check dependencies
+          EXPECTED_LINT_DEPS="\
+              mypy \
+              isort \
+              black \
+              flake8-docstrings \
+              flake8-docstrings-complete \
+              flake8-builtins \
+              flake8-test-docs \
+              pep8-naming \
+              codespell \
+              pylint \
+              pydocstyle \
+              "
+          for EXPECTED_LINT_DEP in $EXPECTED_LINT_DEPS; do
+              DEP_REGEX="deps.*=.*\n(\s.*\n)*(\s.*$EXPECTED_LINT_DEP)"
```

Comment:
A comment to explain the regex would be interesting for understanding

Reply:
Yes that is a good point, I'll add that Monday, the regex basically looks for a line starting with `dep`, then allows for any number of lines that have whitespace at the start followed by a line that has whitespace and then the linter